content/300-guides/200-deployment/301-edge/550-deploy-to-deno-deploy.mdx

```diff
@@ -84,44 +84,41 @@ With the data model in place and your database connection configured, you can no
 ### Steps
 
 ```terminal
-deno run -A npm:prisma db push
+deno run -A npm:prisma migrate dev --name init
 ```
 
-### Result
+The command does two things:
 
-With `prisma db push`, you apply the data model in `schema.prisma` to your database.
+1. It creates a new SQL migration file for this migration
+1. It runs the SQL migration file against the database
 
 At this point, the command has two additional side effects. The command installs Prisma Client and creates the `package.json` file for the project, which includes the `@prisma/client` package as a dependency.
 
-## 3. Generate Prisma Client for the Data Proxy
+## 3. Generate Prisma Client for Prisma Accelerate
 
-Next, generate Prisma Client for the Data Proxy with the `--no-engine` flag. Later, you use the [Data Proxy](/data-platform/classic-projects/data-proxy) to connect to your database over HTTP.
+Next, generate Prisma Client for the Prisma Accelerate with the `--no-engine` flag. Later, you will use [Prisma Accelerate](/data-platform/accelerate/) to connect to your database over HTTP.
 
 ```terminal
 deno run -A --unstable npm:prisma generate --no-engine
 ```
 
 <Admonition>
 
-Prior to Prisma 5.2.0, the `--no-engine` flag is not available. Instead, use the `--data-proxy` flag.
+Prior to Prisma 5.2.0, the `--no-engine` flag is not available. Instead, use the `--accelerate` flag.
 
 ```terminal
-deno run -A npm:prisma generate --data-proxy
+deno run -A npm:prisma generate --accelerate
 ```
```

Comment:
Shouldn't this be `--no-engine` similar to the CFW guide?

Reply:
No. The `--accelerate` is for versions before 5.2.0. `--no-engine` is also documented right before the admonition.